### ISSUE TITLE
Retreive QR-Code on admin

### ISSUE THREAD

**Issue by ukoofred** [cydrobolt/polr#452]
The QR Code is only provided when you shorten an url.

Give a place to the QR Code on the admin page or Stats pages can be a good issue.

**Comment by overint** [comment]
I don't think we need to add it to the admin panel, if you wish to generate a QR code after the fact I think it would be easier to another site generate one.

**Comment by ukoofred** [comment]
I have think it can be usefull for a user to retreive the QR Code after the Link generation.
The QR library is embed, i mean that's an easy improvment of the link_stats.blade.php file.
